src/config/national.js

```diff
@@ -209,24 +245,31 @@ export const NationalConfig = {
         title: 'HISTOGRAM_INFRASTRUCTURE_TITLE'
       }
     },
+    // Default histogram chart height
     height: 150,
-    width: undefined, // Width is dynamically detected from page state
-    callouts: { // Callouts are the vertical bars representing specific values
-      // The country, cluster and soum keys are to match the SoumData.compare response,
-      // which gives averaged values for metrics at the country, cluster and soum levels.
+    // Width is dynamically detected from page state, should be left as 'undefined'
+    width: undefined,
+    // Callouts are the vertical bars representing specific values
+    // The country, cluster and soum keys are to match the SoumData.compare response,
+    // which gives averaged values for metrics at the country, cluster and soum levels.
+    callouts: {
       country: {
+        // Label to display. Supports translation keys.
         label: 'COUNTRY_LEGEND_LABEL',
+        // Color for this metric. Should be a hex value.
         color: '#323349'
       },
       cluster: {
         label: 'CLUSTER_LEGEND_LABEL',
         color: '#B58554'
       },
       soum: {
-        label: undefined, // Overriden by the Soum's name
+        // Overriden by the Soum's name, this label should not be overridden
```

Comment:
We use overridden twice, maybe say `should be left as 'undefined'` like with the histogram width field?